technology/scn3me_subm/tech/README

```diff
@@ -0,0 +1,4 @@
+Techfiles .DRC, .LVS, and .XRC included in this folder for ease of access.
```

Comment:
What about freepdk45 rules and extraction?

Reply:
Freepdk45 rules and extraction are in the Freepdk45 directory. I can add them to the technology directory within OpenRAM and create a README for them if you want. Either way should work